schema/configuration-structure.asn

```diff
@@ -21,6 +103,9 @@ BEGIN
    }
 
    Custom ::= SEQUENCE {
+      customExtName PrintableString, -- Prefixed with a single “X”
+      customExtVersion Version OPTIONAL,
```

Comment:
```suggestion
      name PrintableString, -- Prefixed with a single “X”
      version Version OPTIONAL,
```

It's already implied that it's for a custom extension because of the type that it is in.

Reply:
Also, it seems unnecessary to prefix the name with X. It's just wasting typing/space. We're already in the custom extension section.